src/css/reviews.css

```diff
@@ -1 +1,148 @@
-/* reviews styles */
\ No newline at end of file
+/* reviews styles */
+.reviews-section {
+    padding-bottom: 40px;
+}
+
+.reviews-items {
+    border-radius: 16px;
+    background-color: var(--secondary-bg-color);
+    padding: 20px 20px 40px;
+    height: 278px;
+}
+
+.reviewer-wrap {
+    display: flex;
+    gap: 8px;
+    min-width: 100%;
+    padding: 14px;
+    margin-bottom: 20px;
+    border-radius: 8px;
+    background: var(--substrate-color);
+}
+
+
+.reviewer-img-wrap {
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+.reviewer-txt-wrap {
+    padding: 4px 0;
+}
+
+.reviewer-name {
+    color: var(--secondary-color);
+    font-size: 20px;
+    line-height: 1.2;
+    letter-spacing: -0.4px;
+    margin-bottom: 2px;
+}
+
+.reviewer-status {
+    color: var(--text-color);
+    font-size: 12px;
+}
+
+
+.reviews-text {
+    margin: 0 auto;
+    color: var(--primary-color);
+    font-size: 14px;
+    line-height: 1.71;
+    max-width: 267px;
+    width: 100%;
+    max-height: 120px;
+    overflow-y: auto;
+}
+
+.reviewer-swiper {
+    --swiper-navigation-size: 24px;
+    --swiper-theme-color: var(--border-color);
+    --swiper-navigation-sides-offset: 3px;
```

Comment:
We should do this or globally or at the top of the file and not in the class

Reply:
So these are slider variables. Yaroslav did the same in class. I also change them for different breakpoints.